_pages/about-us/offices/distributed.md

```diff
@@ -36,22 +37,22 @@ title: Distributed
 
 _Before you do anything, read our [best practices for making distributed teams work](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/)._
 
-## How do I&hellip;
+## How do I...
 
 ### Set up my computer?
 
 TTS employees receive a MacBook and an iPhone from GSA, though you may not receive your work phone on your first day.
 
-On your first day at work you should configure things so that your two-factor authentication (2FA) codes &mdash; which you&rsquo;ll occasionally need to connect to GSA&rsquo;s virtual private network (VPN) &mdash; are delivered to your cell phone. If you don&rsquo;t receive your work cell phone on day one, you&rsquo;ll need to use your personal phone. **If you do not configure 2FA you will not be able to access your email from outside of federal buildings.**
+On your first day at work you should configure things so that your two-factor authentication (2FA) codes---which you'll occasionally need to connect to GSA's virtual private network (VPN)---are delivered to your cell phone. If you don't receive your work cell phone on day one, you'll need to use your personal phone. **If you do not configure 2FA you will not be able to access your email from outside of federal buildings.**
```

Comment:
Aidan, does smartify convert three dashes rather than two?

Reply:
Related: do you know of a quick cheat sheet for all the changes?

Reply:
https://kramdown.gettalong.org/syntax.html#typographic-symbols